aikau/src/main/resources/alfresco/forms/controls/BaseFormControl.js

```diff
@@ -541,13 +541,20 @@ define(["dojo/_base/declare",
        * @param {boolean} hasPassedRule Indicated whether or not evaluation of the rules were successful
        */
       autoSetValue: function alfresco_forms_controls_BaseFormControl__autoSetValue(passValue, failValue, hasPassedRule) {
+         // passValue/failValue may not be set to any value in autoSetConfig to achieve positive-/negative-only rule behaviour
          if (hasPassedRule === true)
          {
-            this.setValue(passValue);
+            if (passValue !== undefined)
```

Comment:
Use `typeof === "undefined"` instead, as `undefined` could be overridden by an external library.

_Best practice coding, rather than any great likelihood of it happening_